File: src/test/java/de/umass/util/StringUtilitiesTest.java
package de.umass.util;

import static org.junit.Assert.assertEquals;

public class StringUtilitiesTest {

    @org.junit.Test
    public void md5() {
        String actual = StringUtilities.md5("");
        String expected = "d41d8cd98f00b204e9800998ecf8427e";
        assertEquals(expected, actual);
    }
}

File: src/test/java/com/softartdev/util/StringUtilitiesTest.kt
package com.softartdev.util

import org.junit.Assert.assertEquals
import org.junit.Test

class StringUtilitiesTest {

    @Test
    fun md5() {
        val actual = StringUtilities.md5("")
        val expected = "d41d8cd98f00b204e9800998ecf8427e"
        assertEquals(expected, actual)
    }
}